File: Plots/mvp_plots.js
import d3
import dash
import dash_core_components as dcc
import dash_html_components as html

// Load data csv
d3.csv("mvp_winners.csv").then(function(Data) {

  // Print the Data  
  console.log(Data);

  start = Data.Year(1567)
  end = Data.Year(2016)

  app = dash.Dash()

  app.layout = html.Div(shooting =[

    html.Div(shooting= ''''
        symbol to graph:
    '''),

    dcc.Input( id ='input', value= '', type= 'text')


  ])

  if __name__ = '__main__':
  app.run_server(debug=True)


  //users_year_choice = int (input ( "Type a year betweet 1567 and 2016"))
  //var samples = Data.Year.filter(s == s.users_year_choice)
  //var years = samples.map(function(year) {
    //return year;

  //} 
   
//}


    //data = [{
    // type: 'scatterpolar',
    // r: [39, 28, 8, 7, 28, 39],
    // theta: ['A','B','C', 'D', 'E', 'A'],
    // fill: 'toself'
  //}]
  
  //layout = {
    //polar: {
      //radialaxis: {
        //visible: true,
        //range: [0, 50]
      //}
    //},
    //showlegend: false
  //}
  
  //Plotly.newPlot("myDiv", data, layout)
//})

File: Plots/mvp_plots.py
import dash
import dash_core_components as dcc
import dash_html_components as html
from dash.dependencies import Input, Output
import pandas as pd

Data = pd.read_csv("mvp_winners.csv", usecols = col_list)
col_list = ["Year", "Player", "G", "PTS", "PER", "TS%"]

start = Data["Year"]=1567
end = Data["Year"]=2016
df = Data["Player", "TS&", start, end]

app = dash.Dash()

app.layout = html.Div(shooting =[

    html.Div(shooting= ''''
        symbol to graph:
    '''),

    dcc.Input( id ='input', value= '', type= 'text'),
    html.Div(id= 'output-graph')

  ])
@app.callback (
    Output(component_id = 'output-graph', component_property = 'shooting'),
    [Input (component_id= 'input', component_property= 'value')]
    )
def upgrade_graph(input_data):
    start = Data["Year"] = 1567
    end = Data["Year"] =2016
    df = (input_data, df, start, end)

    return dcc.Graph(
        id = 'example-graph',
        figure={
            'data':[
                {'x': Data["TS&"], 'y': Data["Year"], 'type': 'line','name': input_data},
            ],
            'layout':{
                "title": input_data
            }
        }
    )

if __name__ == '__main__':
    app.run_server(debug=True)

